extensions/ql-vscode/src/qlpack-generator.ts

```diff
@@ -29,6 +30,7 @@ export class QlPackGenerator {
     this.header = "# This is an automatically generated file.\n\n";
 
     this.qlpackFileName = "qlpack.yml";
+    this.sourcePath = join(__dirname, "..", this.folderName);
```

Comment:
`__dirname` refers to the directory that this file is in. Is that what we want here? I think `this.sourcePath` refers to the location where the qlpack is going to be created. It should probably be created in the workspace, not in the extension itself.

Reply:
Yup, currently looking into using `workspace.updateWorkspaceFolders` as Shati suggested above. Sorry, I should've updated the PR! It's been a busy day after ** waves at everything ** but I'm hoping that's the last piece of the puzzle.